Gemfile

```diff
@@ -7,12 +7,14 @@ gem "bcrypt", "3.1.12"
 gem "bootsnap", ">= 1.1.0", require: false
 gem "bootstrap-sass", "3.3.7"
 gem "bootstrap-will_paginate", "1.0.0"
+gem "carrierwave", "1.2.2"
```

Comment:
gem này dùng làm gì nhỉ?

Reply:
gem carrierwave  use to resize image uploaded